build/Helix/RunNugetTests.cmd

```diff
@@ -0,0 +1,22 @@
+robocopy %HELIX_CORRELATION_PAYLOAD% . /s /NP
```

Comment:
Instead of forking this file, update the existing one to take the necessary parameters.